packages/dnb-eufemia/src/components/checkbox/Checkbox.tsx

```diff
@@ -202,7 +202,7 @@ function Checkbox(localProps: CheckboxProps) {
 
   useEffect(() => {
     ref.current.indeterminate = indeterminate
-  }, [indeterminate])
+  }, [indeterminate, ref])
```

Comment:
```suggestion
  }, [indeterminate])
```
https://github.com/dnbexperience/eufemia/pull/3515#discussion_r1593769308